src/frontend/src/app/types/index.ts

```diff
@@ -0,0 +1,5 @@
+export type PlayerData = {
```

Comment:
Name the file PlayerData.ts, since we'll probably want to just keep each type in its own file.